plugin/src/main/scala/io/github/retronym/dependencyanalyzer/DependencyAnalyzer.scala

```diff
@@ -0,0 +1,100 @@
+package plugin.src.main.scala.io.github.retronym.dependencyanalyzer
+
+import plugin.src.main.scala_2_11.io.github.retronym.dependencyanalyzer.Compat
+
+import scala.reflect.io.AbstractFile
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+import scala.tools.nsc.{Global, Phase}
+
+class DependencyAnalyzer(val global: Global) extends Plugin with Compat {
```

Comment:
is this exactly the same code with no changes? If so, can we add a comment to that effect and if not explain where it is different?